exporter/datadogexporter/traces_exporter.go

```diff
@@ -208,7 +208,6 @@ func newTraceAgentConfig(ctx context.Context, params exporter.CreateSettings, cf
 	acfg.AgentVersion = fmt.Sprintf("datadogexporter-%s-%s", params.BuildInfo.Command, params.BuildInfo.Version)
 	acfg.SkipSSLValidation = cfg.ClientConfig.TLSSetting.InsecureSkipVerify
 	acfg.ComputeStatsBySpanKind = cfg.Traces.ComputeStatsBySpanKind
-	acfg.PeerServiceAggregation = cfg.Traces.PeerServiceAggregation
```

Comment:
Why are we removing it ?

Reply:
This was removed since the last datadog-agent version: https://github.com/DataDog/datadog-agent/pull/23284

Reply:
`cfg.Traces.PeerServiceAggregation` was removed in a more recent trace agent version. Should we change this to `acfg.PeerServiceAggregation = cfg.Traces.PeerTagsAggregation` to prevent breaking changes?

Reply:
This is a breaking change, I think we should think about it carefully and align the release of this breaking change with the corresponding change in the Agent 7.54 version

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32823